src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -663,7 +663,9 @@ public void SetRigidity(int rigidity)
 
         if (cost > MutationPoints)
         {
-            gui.UpdateRigiditySlider(intRigidity, MutationPoints);
+            int stepsLeft = MutationPoints / Constants.MEMBRANE_RIGIDITY_COST_PER_STEP;
+            int farthestRigidtyPossible = intRigidity > rigidity ? intRigidity - stepsLeft : intRigidity + stepsLeft;
+            gui.UpdateRigiditySlider(farthestRigidtyPossible, MutationPoints);
             return;
```

Comment:
This return should be removed, right, and rigidity adjusted. Because right now it doesn't cost any ATP for this move and it can't be undone as that code is after this return.